Sources/BraintreeCore/BTAPIClient.m

```diff
@@ -137,31 +137,6 @@ + (BTAPIClientAuthorizationType)authorizationTypeForAuthorization:(NSString *)au
     }
 }
 
-- (instancetype)copyWithSource:(BTClientMetadataSourceType)source
```

Comment:
Using this method prevents the `BTAPIClient` from sharing a cache, we are removing this method so that feature clients can be instantiated with a single `BTAPIClient` and share a cache

Reply:
Is this a breaking change?

Reply:
Shouldn't be?  iirc `-` is private vs `+` public?

Reply:
Ah yes good catch - put back the method and tests and added a next major version note 👍